Bottom: 68924b100a93e4f53bc3e5c917b20d7dc61c58af
Top:    0c645cbe633603e94823152262061516f9d7da71
Author: Ian Kent <ikent@redhat.com>
Date:   2013-04-04 14:33:03 +0800

Refresh of autofs-5.0.7-syncronize-handle_mounts-shutdown.patch

---

diff --git a/include/master.h b/include/master.h
index 824707c..8913740 100644
--- a/include/master.h
+++ b/include/master.h
@@ -115,7 +115,7 @@ int master_mount_mounts(struct master *, time_t, int);
 int master_show_mounts(struct master *);
 extern inline unsigned int master_get_logopt(void);
 int master_list_empty(struct master *);
-int master_done(struct master *);
+void master_done(struct master *);
 int master_kill(struct master *);
 
 #endif
